clients/web/src/views/App.js

```diff
@@ -40,6 +40,7 @@ var App = View.extend({
         this._started = false;
         settings = settings || {};
         if (settings.start === undefined || settings.start) {
+            this.contactEmail = settings.contactEmail;
```

Comment:
We should always store this setting in an instance variable, even if we're not starting the app.